create-project-async/src/portioned_assignment.py

```diff
@@ -18,16 +18,16 @@ def __init__(self, original_assignments: list[dict], multi_pass_prefix: str, sin
         self.multi_pass_labeler_count = multi_pass_labeler_count
 
         # set some properties related to the assignees
-        self.original_role_map: dict[str, AssignmentRole] = {}
+        self.original_role_map: dict[str, str] = {}
         self.labeler_team_member_ids = []
         self.reviewer_team_member_ids = []
         for assignment in self.original_assignments:
             team_member_id = assignment["teamMemberId"]
-            role: AssignmentRole = assignment["role"]
+            role: str = assignment["role"]
 
             # split reviewer and labeler assignees
             # labeler assignees include `LABELER_AND_REVIEWER` role
-            if (role == AssignmentRole.REVIEWER):
+            if (role == AssignmentRole.REVIEWER.value):
```

Comment:
this is fine ya, ini FYI aja (karena aku baru coba-coba jg

bisa compare begini jg sepeertinya
```py
if (AssignmentRole[role] == AssignmentRole.REVIEWER):
  # should also return True if role="REVIEWER"
  pass
```

![image](https://github.com/datasaur-ai/api-client-example/assets/89899945/2fd70b3f-153c-4be5-b0ce-f06119b284ea)

syntax preference aja sih, not sure mana yang lebih python-ic